src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -886,6 +916,72 @@ GrpAb: Z
     return codomain(map_from_torusinvariant_cartier_divisor_group_to_picard_group(v))
 end
 
+@doc raw"""
+    map_from_picard_group_to_class_group(v::AbstractNormalToricVariety)
+
+Return the embedding of the Picard group into the class group of an abstract normal toric variety `v`.
+
+# Examples
+```jldoctest
+julia> p2 = projective_space(NormalToricVariety, 2)
+Normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> map_from_picard_group_to_class_group(p2)
+Map with following data
+Domain:
+=======
+Abelian group with structure: Z
+Codomain:
+=========
+Abelian group with structure: Z
+```
+"""
+@attr GrpAbFinGenMap function map_from_picard_group_to_class_group(v::AbstractNormalToricVariety)
+    f = image(map_from_torusinvariant_cartier_divisor_group_to_class_group(v))[2]
+    g = snf(domain(f))[2] * f
+    return hom(picard_group(v), class_group(v), matrix(g))
+end
+
+
+############################
+# Gorenstein and Picard Index
+############################
+
+@doc raw"""
+    gorenstein_index(v::AbstractNormalToricVariety)
+
+Return the Gorenstein index of an abstact normal toric variety `v`.
+
```

Comment:
We use CLS11 as our standard reference for notation, implementation, algorithms. Hence, I just checked CLS11 and could not find a definition of this index in there - at least not yet. Please correct me if I overlooked it.

If indeed not introduced in CLS11, I would suggest to add a reference to the OSCAR bibliography which introduces this index and link it here. Possibly also add a few explaining words.

Reply:
It is defined under the name "index" in Exercises 8.3.10 and 8.3.11 in CLS11 (only for surfaces, but it works equally well in arbitrary dimension). There are many other references, I can try to find a better one if needed.

Reply:
Thank you for the pointer.